### ISSUE TITLE
[Bug]: Larger sword models shrinks when blocking.

### ISSUE THREAD

**Issue by DelphiDv** [Fuzss/swordblockingmechanics#26]
### Mod Loader (Required)

Forge

### Minecraft Version(s) (Required)

1.20.1

### Mod Version(s) (Required)

v8.0.1

### Minimal Setup (Required)

Yes

### Notes (Required)

Larger sword model/textures gets shrunk down in the blocking pose.
![Screenshot_5](https://github.com/Fuzss/swordblockingmechanics/assets/28810353/1ae300ca-5e34-436c-b0c0-72f0f4a5ccb8)
![Screenshot_4](https://github.com/Fuzss/swordblockingmechanics/assets/28810353/c0716aea-df7a-4295-bf53-919a861b4fce)
![Screenshot_3](https://github.com/Fuzss/swordblockingmechanics/assets/28810353/4a40acf8-d6e1-4236-a1db-72e5dfa6557a)
![Screenshot_2](https://github.com/Fuzss/swordblockingmechanics/assets/28810353/576e8344-73f5-4389-a3b0-1e322aa77e36)

### latest.log (Required)

https://gist.github.com/DelphiDv/73d37b9b5e728bac07c91525b9b8cf4d

**Comment by DelphiDv** [comment]
Extra detail: dosent happen with simple blocking pose on.

**Comment by Fuzss** [comment]
Yeah that's to be expected I guess. I have no clue how the model stuff really works, was happy enough to get it working for the vanilla sword model. So I can only recommend using the simple blocking pose in your case, won't look into it.
